src/crimson/osd/replicated_recovery_backend.cc

```diff
@@ -799,27 +799,11 @@ ReplicatedRecoveryBackend::_handle_pull_response(
   if (pull_info.recovery_progress.first) {
     prepare_waiter = pg.obc_loader.with_obc<RWState::RWNONE>(
       pull_info.recovery_info.soid,
-      [this, &pull_info, &recovery_waiter, &push_op](auto, auto obc) {
+      [this, &pull_info, &recovery_waiter](auto, auto obc) {
+        // Classic can load the obc using a given attrset, Crimson doesn't
+        // load obc using existing attrset yet (consults disk by default instead).
```

Comment:
IIRC, there are cases in which there's no such object in the local disk and we can only construct the `obc` using attrs in the pull response.

Reply:
will be addressed in the (blocked-by) PR and rebased